### ISSUE TITLE
Word after emoji is combined with the emoji in surface.

### ISSUE THREAD

**Issue by couunderbarz** [hideaki-t/igo-python#2]
When igo-python tries to parse a sentence with emoji, a word after the emoji is combined with it in surface.

For example,

```
In [1]: from igo.Tagger import Tagger

In [2]: t = Tagger()

In [3]: s = 'おはようー😳こんにちはー美味しいご飯だよ'

In [4]: for m in t.parse(s):
    print(m)
   ...:     
surface: おはよう, feature: 感動詞,*,*,*,*,*,おはよう,オハヨウ,オハヨー, start=0
surface: ー, feature: 名詞,一般,*,*,*,*,*, start=4
surface: 😳こ, feature: 記号,一般,*,*,*,*,*, start=5
surface: んにちはー, feature: 感動詞,*,*,*,*,*,こんにちは,コンニチハ,コンニチワ, start=7
surface: 美, feature: 名詞,一般,*,*,*,*,*, start=12
surface: 味しいご, feature: 形容詞,自立,*,*,形容詞・イ段,基本形,美味しい,オイシイ,オイシイ, start=13
surface: 飯だ, feature: 名詞,一般,*,*,*,*,ご飯,ゴハン,ゴハン, start=17
surface: よ, feature: 助動詞,*,*,*,特殊・ダ,基本形,だ,ダ,ダ, start=19
surface: , feature: 助詞,終助詞,*,*,*,*,よ,ヨ,ヨ, start=20
```

The correct output is like 

```
surface: おはよう, feature: 感動詞,*,*,*,*,*,おはよう,オハヨウ,オハヨー, start=0
surface: ー, feature: 名詞,一般,*,*,*,*,*, start=4
surface: 😳, feature: 記号,一般,*,*,*,*,*, start=5
surface: こんにちは, feature: 感動詞,*,*,*,*,*,こんにちは,コンニチハ,コンニチワ, start=6
surface: ー, feature: 名詞,一般,*,*,*,*,*, start=11
surface: 美味しい, feature: 形容詞,自立,*,*,形容詞・イ段,基本形,美味しい,オイシイ,オイシイ, start=12
surface: ご飯, feature: 名詞,一般,*,*,*,*,ご飯,ゴハン,ゴハン, start=16
surface: だ, feature: 助動詞,*,*,*,特殊・ダ,基本形,だ,ダ,ダ, start=18
surface: よ, feature: 助詞,終助詞,*,*,*,*,よ,ヨ,ヨ, start=19
```

**Comment by hideaki-t** [comment]
Thank you for your report. I will take a look.

**Comment by hideaki-t** [comment]
@couunderbarz the issue has been fixed, and I published 0.9.8 for that fix.

Actually it affected not only Emojis, but also affected any characters beyond BMP(i.e. characters using surrogate pair in UTF-16)

Thank you again!
